tests/sentry/models/test_organizationmember.py

```diff
@@ -40,6 +40,37 @@ def test_send_invite_email(self):
 
         assert msg.to == ["foo@example.com"]
 
+    def test_send_request_notification_email(self):
```

Comment:
might not hurt to test for the email on the endpoints as well

Reply:
Added